saleor/seo/models.py

```diff
@@ -0,0 +1,14 @@
+from django.core.validators import MaxLengthValidator
+from django.db import models
+
+
+class SeoModel(models.Model):
```

Comment:
`SeoModel` sounds a bit vague, maybe something like `SeoFieldsMixin` would be better?

Reply:
It's not really a mixin, but will think about the better one